src/main/java/no/sikt/nva/pubchannels/handler/FetchJournalByIdentifierAndYearHandler.java

```diff
@@ -0,0 +1,61 @@
+package no.sikt.nva.pubchannels.handler;
+
+import static nva.commons.core.paths.UriWrapper.HTTPS;
+import com.amazonaws.services.lambda.runtime.Context;
+import java.net.HttpURLConnection;
+import java.net.URI;
+import no.sikt.nva.pubchannels.dataporten.DataportenPublicationChannelSource;
+import no.sikt.nva.pubchannels.model.JournalDto;
+import nva.commons.apigateway.ApiGatewayHandler;
+import nva.commons.apigateway.RequestInfo;
+import nva.commons.apigateway.exceptions.ApiGatewayException;
+import nva.commons.core.Environment;
+import nva.commons.core.JacocoGenerated;
+import nva.commons.core.paths.UriWrapper;
+
+public class FetchJournalByIdentifierAndYearHandler extends ApiGatewayHandler<Void, JournalDto> {
+
+    private static final String ENV_API_DOMAIN = "API_DOMAIN";
+    private static final String ENV_CUSTOM_DOMAIN_BASE_PATH = "CUSTOM_DOMAIN_BASE_PATH";
+    public static final String JOURNAL_PATH_ELEMENT = "journal";
+
+    private final transient PublicationChannelSource publicationChannelSource;
+
+    @JacocoGenerated
+    public FetchJournalByIdentifierAndYearHandler() {
+        super(Void.class, new Environment());
```

Comment:
Er ikke dette renere?
`this(new Environment(),DataportenPublicationChannelSource.defaultInstance())`